app/(candidate)/dashboard/people/[[...attr]]/components/person/PersonMap.js

```diff
@@ -0,0 +1,56 @@
+'use client'
+import { usePerson } from '../PersonProvider'
+import Map from '@shared/utils/Map'
+
+export default function PersonMap() {
+  const [person] = usePerson()
+  if (!person) return null
+
+  const places = []
+
+  if (
+    person?.Residence_Addresses_Latitude &&
+    person?.Residence_Addresses_Longitude
```

Comment:
`if (!person)` guard clause should mean you don't need to set `person?` as optional